### ISSUE TITLE
PLY to OBJ

### ISSUE THREAD

**Issue by Giusort** [nschloe/meshio#901]
Hi,
   I used meshio to convert a ply file into an obj file but it loses the colored texture, as you can see in images.
   There is a way to preserve the original texture?

   Thank you!

![ply](https://user-images.githubusercontent.com/49234435/92121504-a4c1e980-edfa-11ea-91e5-efcad1576bd3.PNG)
![obj](https://user-images.githubusercontent.com/49234435/92121518-a8557080-edfa-11ea-9abe-00b17f82fa0e.PNG)

**Comment by nschloe** [comment]
@Giusort Sorry if this comes across rough, but _how_ do you expect anyone to do anything here? The colors aren't there, okay. Maybe a setting in your viewer? _No idea!_

If you suspect that a data set is missing in the output file, which one do you miss? Maybe `meshio-info` can be of help here. Also, you must provide the command and the mesh file that reproduces the issue. Make sure the mesh file isn't too big.

Closing. Feel free to reopen with the above information.

**Comment by nschloe** [comment]
@Giusort It seems that some things aren't clear to you yet. Please join the chat at https://join.slack.com/t/nschloe/shared_invite/zt-cofhrwm8-BgdrXAtVkOjnDmADROKD7A and ping me.

**Comment by Giusort** [comment]
Hi,

   we are working on a 3D photogrammetry reconstruction pipeline and we generate three PLY files that you can download here:

Fused:
https://drive.google.com/file/d/1a51cxuRuUrcITKw-FTqtcdAx1RHIJv4S/view?usp=sharing

Sparse:
https://drive.google.com/file/d/1uf6bp6j-YBVyu9e9WQBCvVbXj3qPuncw/view?usp=sharing

Mesh:
https://drive.google.com/file/d/14thcMfsb6wBmFnN0D6UShYhwFpQDTTCy/view?usp=sharing

There is no problem opening it on MeshLab: they have coloured mesh.

We are trying to use meshio-convert to export PLY in OBJ format; for fused and sparse files we have this error:

![fontana_fused_ply-fontana_fused_obj](https://user-images.githubusercontent.com/49234435/95974548-6b2ac800-0e15-11eb-8f39-c99e9127c655.PNG)

![fontana_sparse_ply-fontana_sparse_obj](https://user-images.githubusercontent.com/49234435/95974572-7251d600-0e15-11eb-9479-6c8ae7a8b602.PNG)

With fontana_mesh.ply the conversion is ok but the file looses mesh:

![fontana_mesh_ply-fontana_mesh_obj](https://user-images.githubusercontent.com/49234435/95974684-97464900-0e15-11eb-9471-b28cbd7faa16.PNG)


Where we are wrong? There is some param to set?

Thank you.

Gius

**Comment by Giusort** [comment]
hank you!
The fix fixes the fused and poor error, but the color loss still remains.
I noticed that the PLY file imported into Meshlab has the vert color function and the OBJ file generated by meshio-convert no.

![fontana_mesh_ply_color_vert](https://user-images.githubusercontent.com/49234435/96130597-6f7be180-0ef8-11eb-8f49-aa7d57ac4360.PNG)

![fontana_mesh_ply_color_mesh](https://user-images.githubusercontent.com/49234435/96130725-9803db80-0ef8-11eb-9261-37e9e2890578.PNG)

So I tried to export in OBJ the original fontana_mesh.ply using Meshlab; it's possible to set the vert color option.

![meshlab_color_vert_option](https://user-images.githubusercontent.com/49234435/96130848-c1246c00-0ef8-11eb-9f19-50bedc86d343.PNG)

Maybe there is a parameter in meshio-convert to export the color vert as well?

Thank you.


Gius
